templates/apollo-graphql-react/files/.gitignore

```
@@ -0,0 +1,3 @@
+node_modules
+.DS_Store
+.idea/*
```

Give git its newline.

```suggestion
.idea/*

```